includes/disk_image_upload.form.inc

```diff
@@ -31,7 +31,8 @@ function islandora_disk_image_upload_form(array $form, array &$form_state) {
     '#default_value' => isset($form_state['values']['disk_image_file']) ? $form_state['values']['disk_image_file'] : NULL,
     '#upload_location' => 'temporary://',
     '#upload_validators' => array(
-      'file_validate_extensions' => $extensions,
+      'file_validate_extensions' => array(),
```

Comment:
The JIRA issue says we should change `exx` to `eo1` at https://github.com/jordandukart/islandora_solution_pack_disk_image/blob/7.x-ISLANDORA-1466/includes/disk_image_upload.form.inc#L21. Did the requirements for this issue change?

Reply:
EnCase files can be chunks bigger than E01s: https://github.com/libyal/libewf/blob/master/documentation/Expert%20Witness%20Compression%20Format%20%28EWF%29.asciidoc#222-ewf-e01, the validator has a regex to allow for these potentialities.

Reply:
@jordandukart right you are, sorry I missed that. I tested uploading a file with the extension `.e02` and `.e99` and both worked. However, the pattern  that `exx` stands for in the allowed extension list is not explained anywhere. Can we add some help text such as "'exx' is a pattern for extensions such as 'e01', 'e02'... 'e25', etc."?